sdks/python/src/opik/evaluation/evaluator.py

```diff
@@ -61,6 +64,7 @@ def evaluate(
         nb_samples=nb_samples,
         workers=task_threads,
         verbose=verbose,
+        project_name=client._project_name if project_name is None else project_name,
```

Comment:
In this particular place you can just pass `project_name=project_name`.
Opik client will use the default string value by itself, if `project_name` is `None` in `TraceData`.